runtime/sema/type.go

```diff
@@ -3219,6 +3219,51 @@ func init() {
 			"Creates a run-time type representing the given static type as a value",
 		),
 	)
+
+	BaseValueActivation.Set(
+		PublicPathType.String(),
+		baseFunctionVariable(
+			PublicPathType.String(),
+			&FunctionType{
+				Parameters: []*Parameter{{
+					Identifier:     "path",
```

Comment:
Naming the parameter "path" looks like it is expected to be a full path, but only an identifier is expected.

```suggestion
					Identifier:     "identifier",
```